sklearn/model_selection/tests/test_search.py

```diff
@@ -294,25 +296,31 @@ def test_grid_search_iid():
     # once with iid=True (default)
     grid_search = GridSearchCV(svm, param_grid={'C': [1, 10]}, cv=cv)
     grid_search.fit(X, y)
-    first = grid_search.grid_scores_[0]
-    assert_equal(first.parameters['C'], 1)
-    assert_array_almost_equal(first.cv_validation_scores, [1, 1. / 3.])
+    assert_equal(grid_search.results_['param_C'][0], 1)
+    assert_array_almost_equal(grid_search.results_['test_split_0_accuracy'][0],
+                              1.)
+    assert_array_almost_equal(grid_search.results_['test_split_1_accuracy'][0],
+                              1. / 3.)
```

Comment:
I meant that the `split_0_accuracy` should be 1, for both `C=1` and `C=10.` since they are separable. So you need not check `grid_search.results_['test_split_0_accuracy'][0]` alone..